tests/Fixer/Whitespace/BlankLineBeforeStatementFixerTest.php

```diff
@@ -306,6 +357,57 @@ public function provideFixWithDeclareCases()
         ];
     }
 
+    /**
+     * @dataProvider provideFixWithDefaultCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithDefault($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'statements' => ['default'],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideFixWithDefaultCases()
+    {
+        return [
+            [
+                '<?php
```

Comment:
This case is doubled and therefore overcomplete. This is because in your next test the `expected` result is the same as this one. When you test with both `input` and `expected` not only will it test that after the `input` is fixed it is changed into the `expected` result, but it also tests that the `expected` result is not changed when provided to the fixer. Two birds one stone ;)

Reply:
That's some awesome magic 😉 removed.